testnets/elystestnet/assetlist.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "elystestnet",
+  "assets": [
+    {
+      "description": "The native token of Elys Network",
+      "denom_units": [
+        {
+          "denom": "uelys",
+          "exponent": 0,
+          "aliases": [
+            "microelys"
+          ]
+        },
+        {
+          "denom": "mscrt",
```

Comment:
I'm guessing this denom_unit is from a copy/paste error?

Reply:
@JeremyParish69 yes will correct that

Reply:
@JeremyParish69 solved